src/release-notes/4-13-0.rst

```diff
@@ -0,0 +1,1552 @@
+   page of the current web interface.
+
+   Feedback is essential for stabilizing this new interface. The modern
+   WebUI is developed in its own repository:
+   https://github.com/freeipa/freeipa-webui.
+   Please report all UI-specific issues or suggestions directly to that
+   project's issue tracker: https://github.com/freeipa/freeipa-webui/issues.
+
+--------------
+
+-  9605: Add support for DoT/DoH for Zero-Trust
+
+      You can now encrypt all DNS queries and responses between DNS clients and IdM DNS servers.
+
+      Administrators can enable DoT during the installation by using the `--dns-over-tls` option. 
+
+      The following options were added to installation utilities for IdM servers, replicas, clients, and the integrated DNS service:
+
+      * `--dot-forwarder` to specify an upstream DoT-enabled DNS server.
+      * `--dns-over-tls-key` and `--dns-over-tls-cert` to configure DoT certificates.
+      * `--dns-policy` to set a DNS security policy to either allow fallback to unencrypted DNS or enforce strict DoT usage.
+      
+      More details are available at: https://freeipa.readthedocs.io/en/ipa-4-12/designs/edns.html
+
+--------------
+
+-  9612 [RFE]: add a tool to quickly detect and fix issues with IPA ID ranges
+
+      With this update, FreeIPA provides the `ipa-idrange-fix` tool. You can use `ipa-idrange-fix` tool to 
+      analyze existing IdM ID ranges, identify users and groups outside these ranges, and propose 
+      to create new `ipa-local` ranges to include them.
+
+      For more information, see the `ipa-idrange-fix(1)` man page. 
+
+--------------
+
+-  9652: IPA requires unique CA certificate subject names
+
+      IPA actively prevented duplicate subjects. This requirement was relaxed with the following limitations: 
+      1) the certificates cannot be added with different trust flags
+      2) the nickname of the CAs must be the same
+      3) an Authority Key Identifier extension should be included in any CA otherwise the chain of trust will not behave as expected
+
+--------------
+
+-  9661 Change the default CA serial number algorithm to random serial numbers
+
+      With this update, automated removal of expired certificates is now enabled by default in FreeIPA on new replicas. 
+      A prerequisite for this is the generation of random serial numbers for certificates using RSNv3, which is now also enabled by default.
+
+      As a result, certificates are now created with random serial numbers and are removed automatically when expired, 
+      after a default retention period of 30 days after expiry. 
+
+--------------
+
+-  9780: [RFE] ipa-client-automount should have an option to include
+   domain of the machine.
+
+--------------
+
+-  9363: Set compat tree and NIS configuration disabled by default 
+   when deploying FreeIPA.
+
+--------------
+
+-  9757 Support full 32-bit ID range space
+
+--------------
+
+- 9842 Add support for LDAP system accounts
```

Comment:
@antoniotorresm  I think this feature deserves proper release notes, what about:

FreeIPA now introduces support for LDAP-based system accounts through a dedicated sysaccount plugin. Administrators can fully manage these accounts using a complete set of CLI commands: add, delete, modify, find, show, enable, and disable, making automation and service integration more consistent and reliable.

We've also enhanced role handling and passsync management across the platform. Roles and baseldap plugins now support system account membership, allowing system accounts to be assigned permissions just like users or hosts.